### PR TITLE
remove autoconnect option

### LINKED ISSUE #4075: Failing to connect to sensors
## Describe the bug

In version 8.0.7 the sync to bluetooth sensors doesn't complete.
This means that you cannot add new sensors or sync temps.

Also: the message `A download is in progress, please try again soon` has been reported as unclear and unhelpful.
Suggest: `Currently downloading temperature logs, please try again soon`

### To reproduce

1. With an existing sensor, wait for it to sync
2. Go to step 1.

### Expected behaviour

Should sync

### Proposed Solution

Remove the `autoconnect` option, it breaks things.

### PR DESCRIPTION
**Pull request by mark-prins**
Fixes #4075 

## Change summary

Removed the option to `autoconnect` as this was causing issues with connecting.
Targeted Master as I thought to push out to Vanuatu ( who are having an issue with sensors, possibly this one? )

## Testing

Steps to reproduce or otherwise test the changes of this PR:

- [ ] Connect to an existing sensor
- [ ] Remove and reconnect one

### Related areas to think about

If there are any general areas of the codebase your changes might have side affects on, mention them here

### CODE REVIEW

**Review by joshxg — APPROVED** [review]
🤷 

dumb!

File: src/bluetooth/BleService.js [hunk 1]
```diff
@@ -55,11 +54,14 @@ class BleService {
   connectAndDiscoverServices = async macAddress => {
     // without the cancel & reconnect further commands
     // were sometimes returning an error: "BleError: Device [mac address] was disconnected"
+    // Note: adding the option `{ autoConnect: true }` prevents the sensors from connecting sometimes :shrug:
     if (await this.manager.isDeviceConnected(macAddress)) {
       await this.manager.cancelDeviceConnection(macAddress);
+    } else {
```
Comment (katherine-sussol) [inline src/bluetooth/BleService.js:60]: image

🤔 ?

**Review by katherine-sussol — APPROVED** [review]
Weeird, that sounds like it was annoying to troubleshoot 😺

File: src/bluetooth/BleService.js (hunk 1 above)
Comment (mark-prins) [reply src/bluetooth/BleService.js]: oops. thanks.
cfa16c82

**Pull request merged**